controllers/db/userroles.controller.ts

```diff
@@ -0,0 +1,45 @@
+import sequelize from "../../src/db/db";
+
+
+export const findUserPermissions = async (attrValue: string, attrType: string) => {
+
+    let userRolesList: any = [];
```

Comment:
You should be able to type this. Any is always a last resort. Just based off the name of your variable this sounds like a list of user roles